addon/components/ember-table/component.js

```diff
@@ -32,8 +32,8 @@ import layout from './template';
 export default Component.extend({
   layout,
   classNames: ['ember-table'],
-
-  'data-test-ember-table': true,
+  attributeBindings: ['dataTestEmberTable:data-test-ember-table'],
+  dataTestEmberTable: true,
```

Comment:
If this is statically setting a `data-test-` property, and it will not longer be stripped in production by `ember-test-selectors` (I don't think it will be stripped?) then a binding isn't needed.

Setting the attribute in `didInsertElement` seems sufficient, ala

```js
  didInsertElement() {
    this._super(...arguments);
    this.element.setAttribute('data-test-ember-table', '');
```